providers/src/airflow/providers/google/cloud/openlineage/mixins.py

```diff
@@ -167,6 +211,21 @@ def _deduplicate_outputs(self, outputs: list[OutputDataset | None]) -> list[Outp
             # if the rowCount or size can be summed together.
             if single_output.outputFacets:
                 single_output.outputFacets.pop("outputStatistics", None)
+
+            # If both outputs contain Column Level Lineage Facet - merge the facets
```

Comment:
When bigquery receives two or more statements in a query it treats it as a SCRIPT type job, and executes each statement in a separate child job. In that case, we extract CLL per child job, parsing the exact part of the whole query that run.

If we submit the following query to BQ, it will execute 3 child jobs and we will receive `my-project.test.new_table` 3 times with different CLL, so in order to get the whole picture we need to merge it.
```
CREATE OR REPLACE TABLE `my-project.test.new_table` AS
SELECT
  id,
  name,
  age
FROM
  `my-project.test.source`;

INSERT INTO `my-project.test.new_table`
SELECT
  id,
  name,
  age
FROM
  `my-project.test.import_table`;

INSERT INTO `new_table`
SELECT
  NULL,
  name,
  age
FROM
  `copy_result`;
```